portal-impl/src/com/liferay/portal/servlet/TunnelServlet.java

```diff
@@ -94,10 +94,19 @@ public void doPost(
 
 			MethodHandler methodHandler = ovp.getValue();
 
+			if (_log.isDebugEnabled()) {
+				_log.debug("Received method handler " + methodHandler);
+			}
+
 			if (methodHandler != null) {
 				MethodKey methodKey = methodHandler.getMethodKey();
 
 				if (!isValidRequest(methodKey.getDeclaringClass())) {
+					if (_log.isWarnEnabled()) {
```

Comment:
@jorgediaz-lr Any specific reason that others are "debug" while this is a "warn"?

Reply:
Hi @dantewang 

Because the other traces are just writing a debug trace with the received/sent information and this one is writing a trace telling that the request was rejected in the same way a warn trace is written here if the user is not autenticated: https://github.com/liferay-core-infra/liferay-portal/blob/7984a1950f43f6d560f07a4c19554c2b85f5a17a/portal-impl/src/com/liferay/portal/servlet/TunnelServlet.java#L59

During my testing this new warn trace "Invalid request" helped me to find this other staging issue: https://issues.liferay.com/browse/LPS-157852

Nevertheless OOTB the default log traces level for TunnelServlet is ERROR, so this warn trace won't be written unless you change the log configuration.